linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator

```diff
@@ -56,17 +56,24 @@ properties ([
             choices: 'read\nrandread\nwrite\nrandwrite',
             name: 'PERF_IO_MODE',
             description: '[FIO] IO_MODE'],
+        [$class: 'ChoiceParameterDefinition',
```

Comment:
move functional params after distro version
put a BISECT_TYPE = boot/functional/performance with a description. Type of bisect that will be performed.

Reply:
Add for each option description that refers to a bisect type its type at the beginning:
example:
[$class: 'ChoiceParameterDefinition',
             choices: 'read\nrandread\nwrite\nrandwrite',
             name: 'PERF_IO_MODE',
            description: '[PERF] [FIO] IO_MODE'],